src/worker/mod.rs

```diff
@@ -648,35 +648,47 @@ impl Worker {
     async fn wf_activation_failed(
         &self,
         run_id: &str,
+        cause: WorkflowTaskFailedCause,
         failure: workflow_completion::Failure,
-    ) -> Result<bool, CompleteWfError> {
+    ) -> Result<WFTReportOutcome, CompleteWfError> {
+        warn!(run_id, failure=?failure, "Failing workflow activation");
```

Comment:
I suppose if something is going wrong it could be... but Shawn did specifically mention users wanting to see this in the logs. I could shift it to be only when we *report* to server which would make sense. I'll do that.